Xamarin.Forms.Platform.Android/Renderers/ShellItemRenderer.cs

```diff
@@ -122,6 +123,7 @@ protected virtual BottomSheetDialog CreateMoreBottomSheet(Action<ShellSection, B
 			// handle the more tab
 			for (int i = 4; i < ShellItem.Items.Count; i++)
```

Comment:
Can take out this magic number in a internal constant?

Reply:
Yeah lets add a constant and a comment explaining